.circleci/config.yml

```diff
@@ -4,169 +4,166 @@ jobs:
     # must be IROHA_HOME:
     working_directory: /opt/iroha
     docker:
-      - image: warchantua/iroha-dev
+      - image: lebdron/iroha-dev
```

Comment:
Can we change the image to use something other than a personal username? Like iroha?